pkg/updateengine/client.go

```diff
@@ -87,6 +94,31 @@ func (c *Client) Close() error {
 	return nil
 }
 
+const (
+	dbusSdName      = "org.freedesktop.systemd1"
+	dbusSdManager   = "/org/freedesktop/systemd1"
+	dbusSdManagerIf = "org.freedesktop.systemd1.Manager"
+)
+
+func (c *Client) disableRPMOSTreeAutoUpdates() error {
+	// for now, just mask the systemd service -- alternatively, also mount
+	// /etc/rpm-ostreed.conf and turn off setting permanently?
+	mgr := c.conn.Object(dbusSdName, dbus.ObjectPath(dbusSdManager))
+
+	err := mgr.Call(dbusMember(dbusSdManagerIf, "MaskUnitFiles"), 0,
+	                [1]string{"rpm-ostreed-automatic.service"}, false, true).Err
+	if err != nil {
+		return err
+	}
+
+	err = mgr.Call(dbusMember(dbusSdManagerIf, "Reload"), 0).Err
```

Comment:
Fine for now, down the line it's probably better to only do this reload if we detect the unit isn't already masked or so.